README.md

```diff
@@ -190,9 +190,16 @@ Viper4Android. It provides a system wide solution, has a convolution engine but
 ### Linux
 #### PulseEffects / EasyEffects
 [PulseEffects / EasyEffects](https://github.com/wwmm/easyeffects) is a Linux module with wide variety of signal
-processing tools including parametric equalizer. From version 5.0.0 onwards, PulseEffects was renamed to EasyEffects and
+processing tools including parametric equalizer.
+  
+From version 6.0.0 onwards, first select the `plugins` tab at
+the bottom of the screen, add the equalizer plugin, and load APO settings by clicking "Load APO Preset" and
+selecting the ParametricEQ.txt file. Pre-amp can be adjusted with the input slider.
```

Comment:
I think it will be not required in 6.1.4

Reply:
It was a recent contribution that will be present in the next release https://github.com/wwmm/easyeffects/pull/1193. But have in mind I was not the one testing it. I do not use APO presets.

Reply:
/cc @ishitatsuyuki

Reply:
Yes, it won't be required in 6.1.4.

Reply:
Well, the current version, 6.1.3, does, so..

Reply:
Can we just claim that for Easyeffects <= 6.1.3 you need to manually adjust the Pre-amp with the input slider ?

Reply:
Done